elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManager.java

```diff
@@ -665,7 +665,7 @@ public void performStreamOperation(LuceneWork singleOperation, IndexingMonitor m
 
 	@Override
 	public void optimize() {
-		// TODO Is there such thing for ES?
+		performStreamOperation( OptimizeLuceneWork.INSTANCE, null, false );
```

Comment:
Yes, HSEARCH-2092 is the one I was talking about related to ElasticsearchIndexWorkVisitor.

This change doesn't do anything indeed as we still have to implement the force merge in ElasticsearchIndexWorkVisitor. That's why I considered it was more a TODO removal than adding a feature. I just put the code here so that the force merge only has to be implemented at one place.

Looks OK to you?